waf_patches/g77.py

```diff
@@ -57,7 +57,7 @@ def get_g77_version(conf, fc):
         conf.fatal('Could not determine the compiler type')
 
     # --- now get more detailed info -- see c_config.get_cc_version
-    cmd = fc + ['-dM', '-E', '-']
+    cmd = fc  # + ['-dM', '-E', '-']
```

Comment:
Nope... Need to patch https://gitlab.com/ita1024/waf/blob/master/waflib/Tools/gfortran.py#L61 but I don't want to waste time on this until we have a decision from @jhunkeler.